tests/test_level.py

```
@@ -0,0 +1,6 @@
+from femtologging import FemtoLevel
+
+
+def test_enum_variants_exposed() -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)